### PR TITLE
refactor: remove exchange/binance switches from whole codebase

### PR DESCRIPTION
**Pull request by Blaumaus**
ASANA Ticket: [UI: refactor - remove exchange/binance switches from whole codebase](https://app.asana.com/0/1125859137800433/1198213973398167/f)

Description: This pull request removes all mentions/switches/etc. of exchanges. Removed all exID's (using 'bitfinex' by default when necessary).
Refactored several components & fixed their prop types.

UI Demonstration:
bfx-hf

### CODE REVIEW

File: src/components/AtomicOrdersTablePanel/AtomicOrdersTablePanel.js [hunk 1]
```diff
@@ -26,43 +26,36 @@ class AtomicOrdersTablePanel extends React.Component {
...
     )
   }
 }
 
 AtomicOrdersTablePanel.propTypes = {
   setFilteredValueWithKey: PropTypes.func.isRequired,
-  atomicOrders: PropTypes.objectOf(PropTypes.object),
-  activeExchange: PropTypes.string,
+  atomicOrders: PropTypes.array, // eslint-disable-line
   activeMarket: PropTypes.objectOf(
```
Comment (alexstotsky) [inline src/components/AtomicOrdersTablePanel/AtomicOrdersTablePanel.js:59]: I think better be used `arrayOf ` here for more precision

File: src/components/Backtester/Backtester.js [hunk 2]
```diff
@@ -137,7 +136,7 @@ Backtester.propTypes = {
       PropTypes.oneOf([null]).isRequired,
     ]),
   ),
-  allMarkets: PropTypes.objectOf(PropTypes.array),
+  markets: PropTypes.array, // eslint-disable-line
```
Comment (alexstotsky) [inline src/components/Backtester/Backtester.js:139]: same as above

File: src/components/ChartPanel/ChartPanel.js [hunk 3]
```diff
@@ -22,28 +22,25 @@ export default class ChartPanel extends React.Component {
...
       currentMarket: PropTypes.shape({
         base: PropTypes.string,
         quote: PropTypes.string,
         restID: PropTypes.string,
       }),
       marketDirty: PropTypes.bool,
-      exchangeDirty: PropTypes.bool,
     }),
-    allMarkets: PropTypes.objectOf(PropTypes.array),
+    markets: PropTypes.array, // eslint-disable-line
```
Comment (alexstotsky) [inline src/components/ChartPanel/ChartPanel.js:37]: same here

File: src/components/PositionsTablePanel/PositionsTablePanel.js [hunk 4]
```diff
@@ -1,36 +1,27 @@
...
+)
 
-  constructor(props) {
-    super(props)
-
-    const { savedState = {} } = props
-    const { currentExchange } = savedState
+PositionsTablePanel.propTypes = {
+  onRemove: PropTypes.func.isRequired,
+  positions: PropTypes.array, // eslint-disable-line
```
Comment (alexstotsky) [inline src/components/PositionsTablePanel/PositionsTablePanel.js:20]: and here

File: src/ui/Icons/FavoriteIcon.props.js [hunk 5]
```diff
@@ -3,7 +3,6 @@ import PropTypes from 'prop-types'
 export const propTypes = {
   onSelect: PropTypes.func,
   prevMarket: PropTypes.object,
-  exchange: PropTypes.string,
   allMarkets: PropTypes.array,
 }
```
Comment (alexstotsky) [inline src/ui/Icons/FavoriteIcon.props.js:7]: All props-types better be moved to a related component for better linting

**Review by alexstotsky — CHANGES_REQUESTED** [review]

**Review by alexstotsky — APPROVED** [review]

**Review by bhoomij — APPROVED** [review]
looks good, minor comment

**Pull request merged**